source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+
+- Easier deployment, driven by user needs.
+- Ability to use the Mattermost Push Notification Service, or MPNS.
+- Apps automatically updated with the latest features, enhancements and security updates.
+
+Again, this is the recommended route which is as simple as downloading the Mattermost mobile application from the corresponding store, entering a URL to point towards your Mattermost server, and logging in.
+
+Custom Apps
+===========
+
+If you desire to customize the applications, or do not want your users downloading the application from the public app stores, you will need to build the apps yourself. 
+
+As the apps are an open source project, it will require a fork, and your team will be responsible for maintaining the fork, as well as keeping the fork up to date with any changes made by Mattermost.
+
+This process can be complicated. It will also greatly increase deployment time, not only initially, but whenever the apps need to be updated. 
+
+We recommend having your development team take a look at the documentation to ensure they understand the scale and requirements of this path. In general, this route will present some challenges, including…
+
+- Obtaining/providing certificates for your custom application
+- Signing your custom applications
+- Distributing your applications via public or private app stores
+
+*******************************************
+Mattermost Push Notification Service (MPNS)
+*******************************************
+
+Receiving notifications on a mobile device is a core value of your mobile deployment.
+
+It also represents a return on your deployment investment through better-connected users. 
+
+A push proxy is a key technology behind notification transmission.  It enables notifications between the server and the mobile application.
+
+Mattermost provides a self-hosted Push Proxy you can deploy, the Mattermost Push Notification Service (MPNS). This is also available via a hosted-by-mattermost option, or hosted push notification service (HPNS)*.
+
+    **Note:** `Only Mattermost Enterprise Edition E10 <https://about.mattermost.com/pricing/>`_ or higher to receive access to our Hosted Push Notification Service (HPNS)
+
+If you will be using the Mattermost applications via the iOS and Android app stores, the HPNS is all you need.
+
+Our Hosted Push Notification Service offers:
+
+- Access to a publicly-hosted Mattermost Push Notification Service (MPNS).
+- An explicit `privacy policy <https://about.mattermost.com/hpns-privacy/>`_ for the contents of unencrypted messages.
+- Encrypted TLS connections:
+    - between HPNS and Apple Push Notification Services
+    - HPNS and Google’s Firebase Cloud Messaging service
+    - HPNS and your Mattermost server
+- Production-level uptime expectations
+- Compatibility with EMM Providers*
+
+\* *When using our publicly available app store applications and the AppConfig standard.*
```

Comment:
I  see no reason to mention this here

Reply:
As mentioned above, this came up as a point of confusion. App Configuration allows management of the app through the EMM provider extending certainly functionality provide by the EMM that isn’t inherent to the app itself. The goal here is to make every effort to discourage wrapping via an EMM, and encourage management with AppConfig.

Reply:
resolved in meeting